drupal8-apache/7.0/usr/local/share/drupal/drupal8_functions.sh

```diff
@@ -0,0 +1,113 @@
+#! /usr/bin/env bash
+
+#####
+# Perform the Drupal 8 build.
+#####
+do_drupal_build() {
+  do_drupal_create_directories
+  do_drupal_permissions
+  do_drupal_legacy_install_script
+}
+
+do_drupal_development_build() {
+  do_drupal_build
+  do_drupal_legacy_development_install_script
+}
+
+#####
+# Perform tasks on container start
+#####
+do_drupal_start() {
+  do_drupal_permissions
+  do_drupal_legacy_install_finalise_script
+}
+
+do_drupal_development_start() {
+  do_drupal_start
+  do_drupal_development_install
+  do_drupal_legacy_development_install_script
+}
+
+#####
+# Create any directories that aren't given by default.
+#####
+do_drupal_create_directories() {
+  mkdir -p "${WEB_DIRECTORY}/sites/default/files"
+}
+
+#####
+# Fix directory permissions.
+#####
+do_drupal_permissions() {
+  if [ "$IS_CHOWN_FORBIDDEN" -ne 0 ]; then
+    # Give the docroot and config directories to the web user.
+    chown -R "${APP_USER}":"${APP_GROUP}" "${WEB_DIRECTORY}"
```

Comment:
Please update to be specific directories if /app/docroot/* is not needed. Any directory owned by the web server can be modified in the event of a security issue being exploited. We want to reduce the surface area of attack as much as possible.